server/modules/monitoring/DeploymentMonitor.js

```diff
@@ -36,8 +38,29 @@ module.exports = {
 
       yield activeDeploymentsStatus.map(
         activeDeploymentStatus => monitorActiveDeploymentStatus(activeDeploymentStatus)
+          .then((status) => {
+            if (status && status !== Enums.DEPLOYMENT_STATUS.InProgress) {
+              let startTime = moment(activeDeploymentsStatus.startTime).toISOString();
+              let endTime = moment.utc().toISOString();
```

Comment:
Would a unix timstamp be a more flexible value to provide to consumers of this message?

Reply:
I always preferred ISO strings as they're just as easily parsed by code and they're human readable as well.

Reply:
👍 Good point, well made. You're right that it's just preference.